lib/index.js

```diff
@@ -159,13 +159,14 @@ EslintValidationFilter.prototype.processString = function processString(content,
   };
 
   if (this.testGenerator) {
+    const result = filteredOutput.results[0];
     let messages = [];
 
     if (filteredOutput.results.length) {
-      messages = filteredOutput.results[0].messages || [];
+      messages = result.messages || [];
```

Comment:
Instead of creating a new array literal when `result.messages` is falsy, we could reassign `messages` to itself since it already references an empty array.